addon/templates/components/power-select-infinity/trigger.hbs

```diff
@@ -0,0 +1,19 @@
+<input type="search"
+  value={{text}}
+  role="combobox"
+  id="ember-power-select-typeahead-input-{{select.uniqueId}}"
+  class="ember-power-select-typeahead-input ember-power-select-search-input ember-power-select-infinity-input"
+  autocomplete="off"
+  placeholder={{placeholder}}
+  oninput={{onInput}}
+  onfocus={{onFocus}}
+  onblur={{onBlur}}
+  disabled={{select.disabled}}
+  onkeydown={{action "handleKeydown"}}
+  onmousedown={{action "stopPropagation"}}>
+{{#if select.loading}}
+    <span class="ember-power-select-typeahead-loading-indicator">
+        <div class="double-bounce1"></div>
+        <div class="double-bounce2"></div>
+    </span>
```

Comment:
can there possibly be a `loadingComponent` option or something, with this html being the default loading component template, so that it can be customized? (e.g. with our usual loading spinner animation)